test/correctness/simd_op_check.cpp

```diff
@@ -1755,8 +1769,14 @@ struct Test {
         check("vnot(v*)", hvx_width/2, ~u16_1);
         check("vnot(v*)", hvx_width/4, ~u32_1);
 
-        check("vsplat(r*)", hvx_width/1, in_u8(0));
-        check("vsplat(r*)", hvx_width/2, in_u16(0));
+        if (is_v62) {
+            // v62 - Broadcasting unsigned 8 bit and 16 bit scalers
+            // check("v*.b = vsplat(r*)", hvx_width/1, in_u8(0));
+            // check("v*.h = vsplat(r*)", hvx_width/2, in_u16(0));
```

Comment:
@aankit-ca - For the time being, get rid of the if condition, essentially keeping the test as it. That means, vectorizing in_u8(0) should lead to a regular "vsplat" even on v62. This is to say that whenever we don't have v62 support we fall back to v60.